lib/Whim/Controller/Display.pm

```diff
@@ -0,0 +1,39 @@
+package Whim::Controller::Display;
+use Mojo::Base 'Mojolicious::Controller';
+use Whim::Mention;
+
+use Readonly;
+Readonly my $OK          => 200;
+Readonly my $ACCEPTED    => 202;
+Readonly my $BAD_REQUEST => 400;
+
+sub display {
+    my $self = shift;
+
+    my $url = $self->param('url');
+
+    warn "url: $url\n";
```

Comment:
Suggestion, not a show-stopper: maybe use [Mojo::Log](https://metacpan.org/pod/Mojo::Log) instead of `warn`. I can see this information staying useful in debugging sessions, so I wouldn't cut it out. But I'm not sure you'd want it on every run.

Reply:
Oops this is of course a lazy development warn that I didn't clean up, so I'll delete it.

But good note about using Mojo::Log in the future instead and letting these sorts of future-useful message-emitters stick around.